### ISSUE TITLE
Missing label on "Find help in" Select element

### ISSUE THREAD

**Issue by tomsedory** [OpenEugene/little-help-book-web#177]
The "Find help in" dropdown (<select>) is missing a label. See screen shot from WAVE extension.

![missing-select-label](https://user-images.githubusercontent.com/75692630/104686293-c50b8a00-56b1-11eb-82a2-eaff9b177fd1.jpg)
